docs/annexes/SPDX-Lite.md

```diff
@@ -1,18 +1,17 @@
-# Annex H: SPDX Lite  
+# SPDX Lite  
 
-## H.1 Definition of the Lite profile <a name="H.1"></a>  
+## Definition of the Lite profile
```

Comment:
```suggestion
## Definition of the Lite profile <a name="H.1"></a>
```

Add back the HTML anchor, as they are also appeared in other files.
Please kindly correct the "H", if it is no longer the right numbering.

Reply:
We should point to the Annex as a file, not subsections.

Reply:
If so, we should remove the HTML anchors in other files as well.

Reply:
If we wait to do everything in this PR, it will never get merged...

Reply:
No prob at all, can do those in separated PRs too.